tests/attribute_tests.cpp

```diff
@@ -57,4 +57,48 @@ BOOST_AUTO_TEST_CASE(StringClone)
     BOOST_CHECK_EQUAL(cs->type(), 1);
 }
 
+BOOST_AUTO_TEST_CASE(IntegerDataConstructor)
+{
+    std::vector<uint8_t> d {0, 0, 0, 20};
```

Comment:
This test value does not cover multibyte conversions. Let's use something more interesting. For example, `{10, 20, 30, 40}`.

Reply:
Same below.